libnmstate/nm/ovs.py

```diff
@@ -45,13 +46,14 @@ class LacpValue:
     OFF = "off"
 
 
-def has_ovs_capability():
-    nm_client = nmclient.client()
-    return nmclient.NM.Capability.OVS in nm_client.get_capabilities()
+def has_ovs_capability(nm_client):
+    return (
+        NM.Capability.OVS in nm_client.get_capabilities() and is_ovs_running()
```

Comment:
Adding `is_ovs_running()` here has to do with this change or is it a piggy-back fix? If the later, could you extract to a separate commit?

Reply:
Nice catch. Indeed, I think you spotted an error. The reason why "is_ovs_running" was not being checked in has_ovs_capability is because we wanted to show up different errors. i.e an user could have openvswitch service running but not the NetworkManager-ovs plugin installed and vice versa.

Reply:
Restored.